### ISSUE TITLE
How To Group <tr>'s and Sort Them as One?

### ISSUE THREAD

**Issue by tdecoy** [javve/list.js#489]
My issue is, each of my "listings" have two table rows. The first row is the one with the cells (<td>'s) that I need to sort by, and the second row is basically just one big cell (e.g. <td colspan="2">Info</td>) that carries some descriptive text for that listing, which I don't need to sort by.

I need these two rows "locked" together, so when the table is sorted, they move as one BASED on sorting by the <td>'s in the first (main) row, of the two.

Now, since a <table> can have have more than one <tbody>, I thought it would make sense to group each "listing" in a <tbody> and have that be each whole listing.

HTML illustration:

> <table>
>     <thead>
>         <tr>
>             <th>Type</th>
>             <th>Qty</th>
>         </tr>
>     </thead>
>     <tbody data-type="listing-1">
>         <tr>
>             <td>Blue widget</td>
>             <td>1</td>
>         </tr>
>         <tr>
>             <td colspan="2">text for listing 1</td>
>         </tr>
>     </tbody>
>     <tbody data-type="listing-2">
>         <tr>
>             <td>Red Widget</td>
>             <td>2</td>
>         </tr>
>         <tr>
>             <td colspan="2">text for listing 2</td>
>         </tr>
>     </tbody>
>     <tbody data-type="listing-3">
>         <tr>
>             <td>Blue widget</td>
>             <td>3</td>
>         </tr>
>         <tr>
>             <td colspan="2">text for listing 3</td>
>         </tr>
>     </tbody>
> </table>

The obvious problem is the fact that I need to give the <tbody> the class="list". It also doesn't work if I just give the table itself that class.

Is there any way to make ListJS do what I want here? I can't be the only one with this issue.

I've already spent days on trying to figure this out, so any help would be GREATLY appreciated. Thanks very much.

**Comment by wolffe** [comment]
I don't think you can have multiple `tbody` elements inside a table, so the structure is not valid.
